reference/library/src/morpheus-master/sass/modules/timeout-alert/_base.scss

```diff
@@ -1,3 +1,10 @@
 #timeout_alert_body{
-
+	z-index: 1001;
+	background: $background-color;
+	position: absolute;
+	top: 10em;
+	left: 10em;
+	width: calc(100% - 20em);
```

Comment:
calc() doesn't have very good Android browser support, I'm not sure this is a big problem as more people move onto newer browsers. Also old copies of IE don't support it (but again not sure we care), http://caniuse.com/#feat=calc